### ISSUE TITLE
Links to Glossary terms do not work in PDF Export

### ISSUE THREAD

**Issue by eufjan** [realtimeprojects/mkdocs-ezglossary#10]
Thanks a lot to this team for developing this plugin, it is a great ressource.

We have encountered a problem in PDF-Export: unfortunately the links to the terms in the Glossary do not work in the PDF Generated via [mkdocs-with-pdf](https://pypi.org/project/mkdocs-with-pdf/) . References to glossary terms are only formatted as links in the pdf but are not "active" links. Ordinary links to other sections work fine in the PDF.

Is there any solution how this can be resolved?

Setup is:
```
plugins:
  - search
  - table-reader
  - include_dir_to_nav
  - ezglossary:
       tooltip: full
  - kroki:
      ServerURL: https://kroki.hzdr.de
      FencePrefix: ''
      HttpMethod: GET
  - with-pdf
```

**Comment by realtimeprojects** [comment]
This should be fixed right now with the [1.6.1 release](https://realtimeprojects.github.io/mkdocs-ezglossary/releases/#v161). Please let me know if you have any further issues with the PDF export.

**Comment by realtimeprojects** [comment]
This is a left-over logging message from my debug session which i removed in [release v1.6.2](https://realtimeprojects.github.io/mkdocs-ezglossary/releases/#v162).

Thanks for your feedback.
